<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditBranch</class>
 <widget class="QMainWindow" name="EditBranch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(95, 151, 235, 255), stop:1 rgba(255, 255, 255, 255));

QPushButton-active {
	position: relative;
	top: 1px;
}

QPushButton{
	background-color: rgb(0, 255, 0);
	color: rgb(255, 255, 255);
	border-radius: 8px;
	border-style: outset;
}

QPushButton:hover {
	color: light-grey;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>258</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="SaveButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="HomeButton">
          <property name="text">
           <string>Main Menu</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>228</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0" rowspan="2" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>198</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="TitleLabel">
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Edit Branch</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>198</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>11</number>
        </property>
        <item row="2" column="1">
         <widget class="QLineEdit" name="BranchValue">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Button">
              <brush brushstyle="LinearGradientPattern">
               <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                <gradientstop position="0.000000000000000">
                 <color alpha="255">
                  <red>95</red>
                  <green>151</green>
                  <blue>235</blue>
                 </color>
                </gradientstop>
                <gradientstop position="1.000000000000000">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </gradientstop>
               </gradient>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="LinearGradientPattern">
               <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                <gradientstop position="0.000000000000000">
                 <color alpha="255">
                  <red>95</red>
                  <green>151</green>
                  <blue>235</blue>
                 </color>
                </gradientstop>
                <gradientstop position="1.000000000000000">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </gradientstop>
               </gradient>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="LinearGradientPattern">
               <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                <gradientstop position="0.000000000000000">
                 <color alpha="255">
                  <red>95</red>
                  <green>151</green>
                  <blue>235</blue>
                 </color>
                </gradientstop>
                <gradientstop position="1.000000000000000">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </gradientstop>
               </gradient>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Button">
              <brush brushstyle="LinearGradientPattern">
               <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                <gradientstop position="0.000000000000000">
                 <color alpha="255">
                  <red>95</red>
                  <green>151</green>
                  <blue>235</blue>
                 </color>
                </gradientstop>
                <gradientstop position="1.000000000000000">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </gradientstop>
               </gradient>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="LinearGradientPattern">
               <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                <gradientstop position="0.000000000000000">
                 <color alpha="255">
                  <red>95</red>
                  <green>151</green>
                  <blue>235</blue>
                 </color>
                </gradientstop>
                <gradientstop position="1.000000000000000">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </gradientstop>
               </gradient>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="LinearGradientPattern">
               <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                <gradientstop position="0.000000000000000">
                 <color alpha="255">
                  <red>95</red>
                  <green>151</green>
                  <blue>235</blue>
                 </color>
                </gradientstop>
                <gradientstop position="1.000000000000000">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </gradientstop>
               </gradient>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Button">
              <brush brushstyle="LinearGradientPattern">
               <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                <gradientstop position="0.000000000000000">
                 <color alpha="255">
                  <red>95</red>
                  <green>151</green>
                  <blue>235</blue>
                 </color>
                </gradientstop>
                <gradientstop position="1.000000000000000">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </gradientstop>
               </gradient>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="LinearGradientPattern">
               <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                <gradientstop position="0.000000000000000">
                 <color alpha="255">
                  <red>95</red>
                  <green>151</green>
                  <blue>235</blue>
                 </color>
                </gradientstop>
                <gradientstop position="1.000000000000000">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </gradientstop>
               </gradient>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="LinearGradientPattern">
               <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                <gradientstop position="0.000000000000000">
                 <color alpha="255">
                  <red>95</red>
                  <green>151</green>
                  <blue>235</blue>
                 </color>
                </gradientstop>
                <gradientstop position="1.000000000000000">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </gradientstop>
               </gradient>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_7"/>
          </item>
          <item>
           <widget class="QGraphicsView" name="graphicsView"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="MissingFieldLabel">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>59</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Button">
              <brush brushstyle="LinearGradientPattern">
               <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                <gradientstop position="0.000000000000000">
                 <color alpha="255">
                  <red>95</red>
                  <green>151</green>
                  <blue>235</blue>
                 </color>
                </gradientstop>
                <gradientstop position="1.000000000000000">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </gradientstop>
               </gradient>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>254</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="LinearGradientPattern">
               <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                <gradientstop position="0.000000000000000">
                 <color alpha="255">
                  <red>95</red>
                  <green>151</green>
                  <blue>235</blue>
                 </color>
                </gradientstop>
                <gradientstop position="1.000000000000000">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </gradientstop>
               </gradient>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="LinearGradientPattern">
               <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                <gradientstop position="0.000000000000000">
                 <color alpha="255">
                  <red>95</red>
                  <green>151</green>
                  <blue>235</blue>
                 </color>
                </gradientstop>
                <gradientstop position="1.000000000000000">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </gradientstop>
               </gradient>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>59</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Button">
              <brush brushstyle="LinearGradientPattern">
               <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                <gradientstop position="0.000000000000000">
                 <color alpha="255">
                  <red>95</red>
                  <green>151</green>
                  <blue>235</blue>
                 </color>
                </gradientstop>
                <gradientstop position="1.000000000000000">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </gradientstop>
               </gradient>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>254</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="LinearGradientPattern">
               <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                <gradientstop position="0.000000000000000">
                 <color alpha="255">
                  <red>95</red>
                  <green>151</green>
                  <blue>235</blue>
                 </color>
                </gradientstop>
                <gradientstop position="1.000000000000000">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </gradientstop>
               </gradient>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="LinearGradientPattern">
               <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                <gradientstop position="0.000000000000000">
                 <color alpha="255">
                  <red>95</red>
                  <green>151</green>
                  <blue>235</blue>
                 </color>
                </gradientstop>
                <gradientstop position="1.000000000000000">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </gradientstop>
               </gradient>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>120</red>
                <green>120</green>
                <blue>120</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Button">
              <brush brushstyle="LinearGradientPattern">
               <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                <gradientstop position="0.000000000000000">
                 <color alpha="255">
                  <red>95</red>
                  <green>151</green>
                  <blue>235</blue>
                 </color>
                </gradientstop>
                <gradientstop position="1.000000000000000">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </gradientstop>
               </gradient>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>120</red>
                <green>120</green>
                <blue>120</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="LinearGradientPattern">
               <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                <gradientstop position="0.000000000000000">
                 <color alpha="255">
                  <red>95</red>
                  <green>151</green>
                  <blue>235</blue>
                 </color>
                </gradientstop>
                <gradientstop position="1.000000000000000">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </gradientstop>
               </gradient>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="LinearGradientPattern">
               <gradient startx="0.000000000000000" starty="0.000000000000000" endx="1.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                <gradientstop position="0.000000000000000">
                 <color alpha="255">
                  <red>95</red>
                  <green>151</green>
                  <blue>235</blue>
                 </color>
                </gradientstop>
                <gradientstop position="1.000000000000000">
                 <color alpha="255">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </gradientstop>
               </gradient>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>*Enter New Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
